### ISSUE TITLE
Feature request: expanding available platform installations

### ISSUE THREAD

**Issue by VGrol** [jellyfin/jellyfin-ffmpeg#5]
With ffmpeg <4.1 being required for jellyfin, 
It'd be lovely if jellyfin-ffmpeg could become more of a universal solution to the platform.

For entirely selfish reasons, I'd love to see support for Arch, as ffmpeg 4.0 conflicts with a lot of repository packages which are prebuilt.
Having it as a dependency for ffmpeg in general would be much more suitable, one only needs to look at the package pages for users that can't figure out the ffmpeg version requirement.

I'm not entirely sure if this is out of scope for this project, but the current solutions on other platforms are a bit frustrating in general.

**Comment by joshuaboniface** [comment]
I agree with this actually, it makes sense to have a single copy and build from it for every platform.

I've already integrated the Dockerfile that the Jellyfin Docker containers use into here, so if you or anyone can provide build infra for the various other distros (using Dockerfiles like the Debian/Ubuntu builds do) we can add that in!

**Comment by dkanada** [comment]
@VGrol since we use a recent version now would you consider this issue resolved?

**Comment by VGrol** [comment]
@dkanada 
I think it'd depend on the philosophy.
The issue is resolved in the sense that it is functional on Arch. 

The problem is going to happen again whenever FFMPEG updates, since Arch tends to move relatively quickly onto the new version. Until we somehow get a modular code base which supports all future FFMPEG upgrades (lol), a jellyfin-ffmpeg expansion for Arch would probably be better to prevent future breakage.

**Comment by JustAMan** [comment]
Not anymore, we don't require <4.1 now. We're wanting >=4.0 IIRC :)

**Comment by Shadowghost** [comment]
For Arch there now exists an AUR package: https://aur.archlinux.org/packages/jellyfin-ffmpeg

Nonetheless I'll keep the issue opened since there are other distros/packaging systems not supported by our ffmpeg which we're glad to get community contributed build support
